File: AINewsAPI.Domain/Entities/NewsItem.cs
namespace AINewsAPI.Domain.Entities
{
    public class NewsItem
    {
        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string Url { get; set; } = string.Empty;
        public DateTime PublishedAt { get; set; }
    }
}

File: AINewsAPI.Frontend/src/components/NewsItem.js
import React from 'react';

function NewsItem({ item }) {
  return (
    <div className="news-item">
      <h2>{item.title}</h2>
      <p>{item.description}</p>
      <a href={item.url} target="_blank" rel="noopener noreferrer">Read more</a>
    </div>
  );
}

export default NewsItem;
